recipes/package.rb

```diff
@@ -1,6 +1,6 @@
-if(node[:ruby_installer][:package_name])
-  if(node[:ruby_installer][:package_name] == true)
-    case node.platform_family
+if node['ruby_installer']['package_name']
+  if node['ruby_installer']['package_name'] == true
+    case node['platform_family']
```

Comment:
All the duplicated logic here makes me think this could be simplified. The only value I see from doing each of these blocks independently is the specific error message. 

The error handling, however, really reduces the out-of-the-box usefulness of this cookbook. Specifically, any platform that isn't debian, fedora, or rhel will blow up running this recipe.